riotfile.py

```diff
@@ -2369,6 +2369,26 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "requests": "==2.28.1",  # specific version expected by tests
             },
         ),
+        Venv(
+            name="openai",
+            command="pytest {cmdargs} tests/contrib/openai",
+            pys=select_pys(min_version="3.8"),
```

Comment:
```suggestion
```

Reply:
right ? I think otherwise we'd create a venv here that will run, but then won't have numpy ?

(I haven't reviewed the test code, so if I am wrong, just resolve this)